src/chaplin/views/view.coffee

```diff
@@ -194,7 +194,7 @@ module.exports = class View extends Backbone.View
   #   e.g.
   #   @delegate('click', 'button.confirm', @confirm)
   delegate: (eventName, second, third) ->
-    if Backbone.utils
+    if Backbone.utils and Backbone.utils?.delegate
```

Comment:
This is still doing the same thing. 

I would move this change to the next line:

```coffee
if Backbone.utils
  return Backbone.utils.delegate?(this, eventName, second, third)
```